chat-plugins/trivia.js

```diff
@@ -198,7 +198,7 @@ var Trivia = (function () {
 	};
 
 	Trivia.prototype.startGame = function (output) {
-		if (this.phase !== 'signup') return output.sendReply("There is no trivia game in its signup phase.");
+		if (this.phase !== 'signup') return output.sendReply("There is no trivia game to join.");
```

Comment:
Oops, you're right. That's a goof on my part. I will edit it to say start instead of join.